beacon-chain/blockchain/metrics.go

```diff
@@ -204,18 +193,6 @@ func reportEpochMetrics(state *stateTrie.BeaconState) {
 	}
 }
 
-// This captures metrics for block sent time by subtracts slot start time.
-func captureSentTimeMetric(genesisTime uint64, currentSlot uint64) error {
```

Comment:
this is pretty useful and I know we have use this for debugging  so not sure if we should remove it.
cc @terencechain

Reply:
I think it's ok to leave this on until beta. 

We plan to do a fine comb of metrics then.